obspy/io/nordic/core.py

```diff
@@ -824,7 +869,10 @@ def _read_picks(tagged_lines, new_event, nordic_format='UKN', **kwargs):
                     tagged_lines[tag], key=lambda tup: tup[1])])
         except KeyError:
             pass
-    header = sorted(tagged_lines['7'], key=lambda tup: tup[1])[0][0]
+    try:
+        header = sorted(tagged_lines['7'], key=lambda tup: tup[1])[0][0]
+    except IndexError:  # Compact catalog file without picks?!
+        pass
```

Comment:
This might be OK but indexing is used 4 times on that line, so it might or nmight not be too "catch all" of an exception handling? Up to you though, I don't know the details here too much.